ios/FluentUI/Command Bar/CommandBarButton.swift

```diff
@@ -73,6 +74,22 @@ class CommandBarButton: UIButton {
         updateState()
     }
 
+    override func layoutSubviews() {
+        super.layoutSubviews()
+
+        guard let accentImage = item.accentImage, let imageView = imageView else {
+            return
+        }
+        
+        let iconImageFrame = imageView.frame
+
+        accentImageView = UIImageView(image: accentImage)
+        if let accentImageView = accentImageView {
+            insertSubview(accentImageView, belowSubview: imageView)
```

Comment:
we shouldn't be modifying the view hierarchy in layoutSubviews. We should fix up the hierarchy when accentImage is set and then either can use layoutSubviews or just add constraints to keep them positioned in the same place. But layout happens often and manipulating the view hierarchy should only happen when needed

Reply:
Pulled the view hierarchy manipulation out of layoutSubviews.